helpers/wrapper/test/Wrapper-unit.js

```diff
@@ -74,7 +74,8 @@ contract('Wrapper Unit Tests', async accounts => {
   }
 
   async function setupMockSwap() {
-    let swapTemplate = await Swap.new()
+    let mockTokenRegistry = await MockContract.new()
```

Comment:
I think you can just pass in an `accounts[i]` as the mock token registry as it isn't actually being used?